enterprise-suite/values.yaml

```diff
@@ -3,6 +3,7 @@
 #
 consoleUIConfig:
   logo: ""
+  isMonitorEditEnabled: false
```

Comment:
I imagine you named this the same as the field in our env, to minimize confusion – which is a worthy goal. This is a bit of a mouthful though for a customer. Maybe it doesn't matter, since we don't think (m)any customers will be leveraging this. If they do, we probably want a more concise field.

Reply:
yeah.  I use this name is just to be the same name as our env field.  If you want, I can change both UI config field and env field to shorter name like 'enableMonitorEdit' .

Reply:
If customers start using this, maybe we'll change it; or might end up just being a non-issue